PMSCore/Utils/UserManager.cs

```diff
@@ -0,0 +1,138 @@
+                    "\nUsername: user1" +
+                    "\nPassword: *****" +
+                    "\n----------------------------------------------------------------------" +
+                    "\nUser would enter username and password here to be sent to back end.");
+
+                //*
+                // response = StateManager.ValidateUser(username, userpassword);
+
+                // if(response[0] == true && response[1] == true){
+                //      this._user = StateManager.LoadUserProfile(username);
+                //  }
+                //  elif(response[1] == false)
+                //  {
+                //      _logger.LogWarning("Invalid Password for User Account.{username}", username);
+                //  }
+                //  else
+                //  {
+                //      this.Register();
+                //  }
+                //*
+
+                // Simulates user login by creating a test user and assigning it a new Permission_Object.
+                _currentUser = new User_Tester
+                    (
+                    new LoggerFactory()
+                    .CreateLogger("User_Tester"),
+                    new Permission_Object(2),
+                    "J-Doe"
+                    );
+            }
+
+            return _currentUser != null; // Return true if user is successfully created.
+        }
+
+        /// <summary>
+        /// Simulates the registration process for a new user.
+        /// </summary>
+        /// <returns>True if registration succeeds, false otherwise.</returns>
+        public bool RegisterNewUser()
+        {
+            _logger.LogDebug("Register New User page loaded.");
+            // Menu.DisplayRegistrationMenu();
+
+            Console.WriteLine("Simulating Registration process..." +
+                "\n----------------------------------------------------------------------" +
+                "\n To Register please enter your username, followed by your password." +
+                "\nUsername: user1" +
+                "\nPassword: *****" +
+                "\n----------------------------------------------------------------------");
+
+            //userID = PMSUtils.Sanitize(Console.NextLine());
+            //userpassword = PMSUtils.Sanitize(Console.NextLine());
+
+
+            //if(StateManager.AuthenticateNewUser(userID, userpassword))
+            //{
+            //_currentUser = StateManager.LoadUserProfile(userID);
+            //}
+
+
+            // Simulates user registration by creating a test user and passing in a test Permission_Object.
+            _currentUser = new User_Tester
+                (
+                new LoggerFactory()
+                .CreateLogger("User_Tester"),
+                new Permission_Object(2),
+                "J-Doe"
+                );
+
+            return _currentUser != null; // Return true if user is successfully created.
+        }
```

Comment:
Register should receive a user object, if it exists in the list current_user returns an error else adds it to the list